<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShaderLabs</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../shaderlab.qrc">
    <normaloff>:/ico/img/1303588878_laboratory.png</normaloff>:/ico/img/1303588878_laboratory.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow::separator {
     background:  rgb(220, 220, 220);
     width: 2px; /* when vertical */
     height: 2px; /* when horizontal */
 }

 QMainWindow::separator:hover {
     background:  rgb(170, 170, 170);
 }</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="centralLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>743</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLoad_Project"/>
    <addaction name="actionSave_project"/>
    <addaction name="actionSave_as_project"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Object"/>
    <addaction name="separator"/>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpenCode"/>
    <addaction name="separator"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionSaveAll"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menu_Models">
     <property name="title">
      <string>&amp;Models</string>
     </property>
    </widget>
    <addaction name="actionWireframe"/>
    <addaction name="menu_Models"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Build">
    <property name="title">
     <string>&amp;Build</string>
    </property>
    <addaction name="actionRunAll"/>
   </widget>
   <widget class="QMenu" name="menuSettin_gs">
    <property name="title">
     <string>Settin&amp;gs</string>
    </property>
    <widget class="QMenu" name="menu_Geometry_Shader">
     <property name="title">
      <string>&amp;Geometry Shader</string>
     </property>
     <addaction name="actionChange_output_primitive"/>
    </widget>
    <addaction name="menu_Geometry_Shader"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menu_Build"/>
   <addaction name="menuSettin_gs"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_Project"/>
   <addaction name="separator"/>
   <addaction name="actionNewFile"/>
   <addaction name="actionOpenCode"/>
   <addaction name="actionSaveFile"/>
   <addaction name="separator"/>
   <addaction name="actionRunAll"/>
  </widget>
  <widget class="QDockWidget" name="dockRenderWidget">
   <property name="minimumSize">
    <size>
     <width>186</width>
     <height>160</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>   Render Area</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockRenderWidgetContents">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>9</number>
       </property>
       <item>
        <widget class="QWidget" name="optionsContents" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="saveImageButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../shaderlab.qrc">
              <normaloff>:/ico/img/1303587978_3floppy_unmount.png</normaloff>:/ico/img/1303587978_3floppy_unmount.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="originButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../shaderlab.qrc">
              <normaloff>:/ico/origin</normaloff>:/ico/origin</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckbLightRotation">
            <property name="text">
             <string>Light Rotation</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="optionsContents2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="objectsVisibility">
            <property name="text">
             <string>Old Objects</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockOutPutWidget">
   <property name="minimumSize">
    <size>
     <width>97</width>
     <height>121</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Application Output</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockOutPutWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="outputTextBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpenCode">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../shaderlab.qrc">
     <normaloff>:/ico/img/1302039979_Folder.png</normaloff>:/ico/img/1302039979_Folder.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open shader...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../shaderlab.qrc">
     <normaloff>:/ico/img/1302043859_Log Out.png</normaloff>:/ico/img/1302043859_Log Out.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRunAll">
   <property name="icon">
    <iconset resource="../shaderlab.qrc">
     <normaloff>:/ico/img/1302039654_buttons_20.png</normaloff>:/ico/img/1302039654_buttons_20.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="icon">
    <iconset resource="../shaderlab.qrc">
     <normaloff>:/ico/img/1303587978_3floppy_unmount.png</normaloff>:/ico/img/1303587978_3floppy_unmount.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="icon">
    <iconset resource="../shaderlab.qrc">
     <normaloff>:/ico/img/1303588715_document-new.png</normaloff>:/ico/img/1303588715_document-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New shader...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save shader &amp;as ...</string>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="icon">
    <iconset resource="../shaderlab.qrc">
     <normaloff>:/ico/img/1303588015_save_all.png</normaloff>:/ico/img/1303588015_save_all.png</iconset>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionWireframe">
   <property name="icon">
    <iconset resource="../shaderlab.qrc">
     <normaloff>:/ico/img/wireframe_icon.jpg</normaloff>:/ico/img/wireframe_icon.jpg</iconset>
   </property>
   <property name="text">
    <string>Wireframe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionChange_output_primitive">
   <property name="text">
    <string>Change default primitives...</string>
   </property>
  </action>
  <action name="actionLoad_Project">
   <property name="icon">
    <iconset resource="../shaderlab.qrc">
     <normaloff>:/ico/img/Project-icon.png</normaloff>:/ico/img/Project-icon.png</iconset>
   </property>
   <property name="text">
    <string>Open project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSave_as_project">
   <property name="text">
    <string>Save as ShaderLabs Project...</string>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="text">
    <string>Save project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionNew_Object">
   <property name="text">
    <string>New Object</string>
   </property>
  </action>
  <action name="actionPoint">
   <property name="text">
    <string>Point</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../shaderlab.qrc"/>
 </resources>
 <connections/>
</ui>
